api/src/main/resources/workbench.yaml

```diff
@@ -71,6 +71,21 @@ parameters:
     format: int64
     required: true
     description: Data set ID
+  workspaceNamespace:
+    in: path
+    name: workspaceNamespace
+    type: string
+    required: true
+    description: The workspace namespace, aka Google Cloud project ID.
```

Comment:
It would help to indicate if this the firecloud workspace namespace or an AOU one.